software_homework_1/task_1.py

```diff
@@ -0,0 +1,100 @@
+"""
+
+TASK 1
```

Comment:
4/5 - 1 mark deducted for use of print statements instead of return (class definitions should never have print statements). This doesn't mean you won't be able to format things nicely, but most formatting is done outside of a class function definition.